src/xml_interface.cpp

```diff
@@ -9,7 +9,7 @@
 namespace openmc {
 
 std::string
-get_node_value(pugi::xml_node node, const char *name)
+get_node_value_str(pugi::xml_node node, const char* name)
```

Comment:
Why did you need to make this function?  If it's just because of the lowercasing, I would rather there was an optional argument to stop the lowercasing (we need it for `<name>` tags anyway).

Reply:
Yeah, there were some cases where we were lowercasing and others where we were not. I'll combine these two functions into a single one with optional arguments and only use lowercasing where it is necessary.